book/website/community-handbook/pathways.md

```diff
@@ -16,43 +16,47 @@ This can overwhelm users, making it difficult to find relevant content and under
 This user experience risks discouraging community members from effectively using *The Turing Way* and adopting the good practices we champion.
 
 To tackle this issue, we've introduced the 'Pathways' feature.
-Drawing on the concepts of {ref}Personas and Pathways<pd-persona>, this is a Python package implementation, designed to improve content discovery (pathways) by preventing users (personas) from being overwhelmed with information that may not be immediately relevant.
+Drawing on the concepts of [Personas and Pathways](#pd-persona), this is implemented as a [MyST plugin](https://github.com/the-turing-way/myst-curation), designed to improve content discovery (pathways) by preventing users (personas) from being overwhelmed with information that may not be immediately relevant.
 
 ```{figure} ../figures/open-development-path.jpg
 ---
 height: 400px
 name: open-development-path
-alt: Here we have a group of people from different areas who are talking about topics of their interests - a good way to show how pathways can help not just the people from the group it is prepared for, but others who are interested in the field.
+alt: >
```

Comment:
I've not seen this syntax used before - is it a way of enabling multi-line alt-texts? Should we document in the style guide?